pkg/systemd/services.html

```diff
@@ -92,42 +92,30 @@
       <div class="panel-heading">{{Unit.Description}}</div>
       <div class="list-group">
         <div class="list-group-item">
-          <table>
-            <tr>
-              <td>
-                <span>{{Unit.ActiveState}}</span>
-                ({{Unit.SubState}})
-                <br/>
-                {{Since}}
-              </td>
-              <td>
-                {{{UnitButton}}}
-              </td>
-            </tr>
+          {{#HasLoadError}}
+            <div class="alert alert-danger">
+              <span class="pficon pficon-error-circle-o"></span>
+              <strong>{{Unit.LoadState}}</strong> Reason: {{LoadError}}
+            </div>
+          {{/HasLoadError}}
+          <table class="form-table-ct">
+            <tr><td class="top-right">Status</td><td>{{Unit.ActiveState}} ({{Unit.LoadState}}) {{{UnitButton}}}</td></tr>
+            {{#Since}}
+              <tr><td class="top-right">Running Since</td><td>{{Since}}</td></tr>
```

Comment:
This is a new one, though. Please add the `translate` property to the containing `<td>`.

Reply:
Also, it seems the value already contains the word "since", so it's redundant.

Reply:
Added translate to all td's and remove the redundant "since" from the date string and kept it only in the label.